Dizzy/Screens/Login/LoginVC.swift

```diff
@@ -0,0 +1,228 @@
+        loginContainerView.addSubviews([titleLabel, subtitleLabel, loginSelectionView,
+                                        appInfoView, dizzyLogoImageView, enterAsAdminButton])
+        
+    }
+    private func layoutViews() {
+        
+        layoutCloseButton()
+        layoutLoginContainerView()
+        
+        layoutTitleLabel()
+        layoutSubtitleLabel()
+        layoutLoginSelectionView()
+        layoutAppInfoView()
+        
+        layoutDizzyLogo()
+        layoutEnterAsAdminButton()
+    }
+    
+    private func layoutCloseButton() {
+        closeButton.snp.makeConstraints { closeButton in
+            
+            closeButton.top.equalTo(self.view.snp.topMargin).offset(Metrics.padding)
+            closeButton.trailing.equalToSuperview().offset(-Metrics.doublePadding)
+        }
+    }
+    
+    private func layoutLoginContainerView() {
+        loginContainerView.snp.makeConstraints { loginContainerView in
+            
+            loginContainerView.top.equalTo(closeButton.snp.bottom).offset(Metrics.padding)
+            loginContainerView.leading.trailing.equalToSuperview()
+            loginContainerView.bottom.equalToSuperview().offset(Metrics.doublePadding)
+        }
+    }
+    
+    private func layoutTitleLabel() {
+        titleLabel.snp.makeConstraints { titleLabel in
+            titleLabel.top.equalToSuperview().offset(Metrics.padding)
+            titleLabel.leading.trailing.equalToSuperview().offset(Metrics.padding)
+        }
+    }
+    
+    private func layoutSubtitleLabel() {
+        subtitleLabel.snp.makeConstraints { subtitleLabel in
+            subtitleLabel.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding)
+            subtitleLabel.leading.trailing.equalToSuperview().offset(Metrics.padding)
+        }
+    }
+    
+    private func layoutLoginSelectionView() {
+        loginSelectionView.snp.makeConstraints { loginSelectionView in
+            loginSelectionView.top.equalTo(subtitleLabel.snp.bottom)
+            loginSelectionView.leading.trailing.equalToSuperview()
+        }
+    }
+    
+    private func layoutAppInfoView() {
+        appInfoView.snp.makeConstraints { appInfoView in
+            appInfoView.top.equalTo(loginSelectionView.snp.bottom)
+            appInfoView.leading.trailing.equalToSuperview()
+        }
+    }
+    
+    private func layoutDizzyLogo() {
+        dizzyLogoImageView.snp.makeConstraints { dizzyLogoImageView in
+            dizzyLogoImageView.top.equalTo(appInfoView.snp.bottom).offset(2 * Metrics.doublePadding)
+            dizzyLogoImageView.leading.trailing.equalToSuperview()
+        }
+    }
+    
+    private func layoutEnterAsAdminButton() {
+        enterAsAdminButton.snp.makeConstraints { enterAsAdminButton in
+            enterAsAdminButton.top.equalTo(dizzyLogoImageView.snp.bottom).offset(Metrics.doublePadding)
+            enterAsAdminButton.leading.trailing.equalToSuperview()
+            enterAsAdminButton.height.equalTo(40)
```

Comment:
please move height to be a class variable

Reply:
Done!